src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -111,6 +168,65 @@
                     "description": "Milliseconds since last update from peer"
                 }
             }
+        },
+        "self_test_report": {
+            "id": "self_test_report",
+            "description": "Self test single test report",
+            "properties": {
+                "disk_test_stoptime": {
+                    "type": "long",
+                    "description": "time to finish stub disk test"
+                },
+                "network_test_stoptime": {
+                    "type": "long",
+                    "description": "time to finish stub network test"
+                },
+                "failures": {
+                    "type": "boolean",
+                    "description": "True if any failures occurred"
+                }
+            }
+        },
+        "self_test_node_status": {
+            "id": "self_test_node_status",
+            "description": "Self test status",
```

Comment:
"report" and "status" are pretty much synonyms, in these two  API endpoint.  Maybe "result" is a better noun to express the idea of something that comes out at the end, vs. something that is mutable and will change as time goes on.

Or... do we need two endpoints?  The status endpoint includes the report anyway.

Reply:
type removed, resolving